### PR TITLE
NetHui Tickets Issuance page

### PR DESCRIPTION
**Pull request by rafaelbernard**
- You can opt to use Membership
- Organisation is optional

### CODE REVIEW

File: event-ticket.html [hunk 1]
```diff
@@ -0,0 +1,440 @@
+<!doctype html>
```
Comment (ehsan-fj) [inline event-ticket.html:1]: Can't we get Fullname and Email by asking user to disclose their attribute? that's the thing we want to achieve by DI, right?

**Review by ehsan-fj — CHANGES_REQUESTED** [review]

**Comment by ehsan-fj** [comment]
at the beginning I was a bit confused! So don't you think it's better to not show that fill the form section till the user scan the QR Code? Because I though I can fill the from form and issue the ticket!
Also it's better to not allow the user to edit their details as they are already verified and we don't to change them?

**Comment by ehsan-fj** [comment]
Also I think we don't need to store full name and email in scheme as they can be obtained from other issuers.

**Comment by rafaelbernard** [comment]
> at the beginning I was a bit confused! So don't you think it's better to not show that fill the form section till the user scan the QR Code? Because I though I can fill the from form and issue the ticket!
> Also it's better to not allow the user to edit their details as they are already verified and we don't to change them?

I left open because someone could buy a ticket for someone else. I think I will kepp this option although we are not storing name and email (we can point that the back-end would save whom the access pass belongs to).

And I left an option to buy a ticket without the membership credential (will need to pay full price).

**Comment by ehsan-fj** [comment]
> > at the beginning I was a bit confused! So don't you think it's better to not show that fill the form section till the user scan the QR Code? Because I though I can fill the from form and issue the ticket!
> > Also it's better to not allow the user to edit their details as they are already verified and we don't to change them?
> 
> I left open because someone could buy a ticket for someone else. I think I will kepp this option although we are not storing name and email (we can point that the back-end would save whom the access pass belongs to).
> 
> And I left an option to buy a ticket without the membership credential (will need to pay full price).

for buying ticket without IRMA

**Comment by ehsan-fj** [comment]
sorry closed by mistake!

**Review by ehsan-fj — APPROVED** [review]

**Pull request merged**